internal/rbd/controllerserver.go

```diff
@@ -1050,6 +1050,9 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 		return nil, status.Error(codes.InvalidArgument, "capacityRange cannot be empty")
 	}
 
+	// Get the access Block type
+	isBlock := req.GetVolumeCapability().GetBlock() != nil
```

Comment:
nodeExpansion can be set here no need to have a separate variable `isBlock`

Reply:
done.